raster-foundry/raster-foundry
# Improve image overlay clipping

## Overview

This updates the method used in the image overlay service and iterates over all outer polygons (ignores inner rings) to generate clipping shapes

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Note

This isn't ideal. It would be much cleaner to be able to directly use the data-footprint GeoJson rather than having to go through a manual transform. http://bl.ocks.org/d3noob/9211665 looks like it may be a good head-start.

## Testing Instructions

 * Verify that a scene with a multi-polygon data-footprint has the thumbnail displayed correctly
 * Verify that a scene with a single polygon data-footprint has the thumbnail displayed correctly
